src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java

```diff
@@ -409,6 +409,12 @@ private FulfilmentRequest createFulfilmentRequestPayload(
     Product product = findProduct(fulfilmentCode, deliveryChannel, region);
 
     if (deliveryChannel.equals(Product.DeliveryChannel.POST)) {
+      if (caze.isHandDelivery()) {
+        log.warn("This fulfilment is for hand delivery only and so it must not be sent by post");
```

Comment:
Re should this be a warning or info logging. I guess the key question is, should this happen in production do we want to be called to investigate? 
I'm inclined to think that this should be logged at info level.

Reply:
I agree